cmd/tailscale/cli/serve.go

```diff
@@ -171,6 +171,15 @@ func (e *serveEnv) getLocalClientStatus(ctx context.Context) (*ipnstate.Status,
 	return st, nil
 }
 
+// parsePort parses a port string into a uint16.
+func (e *serveEnv) parsePort(portStr string) (uint16, error) {
```

Comment:
the receiver is never used. Do you plan to make it used? For what?

Reply:
https://github.com/tailscale/tailscale/blob/a82c3cae977a3e9f86ab3ddcb85198491d052b1a/cmd/tailscale/cli/serve.go#L231

https://github.com/tailscale/tailscale/blob/a82c3cae977a3e9f86ab3ddcb85198491d052b1a/cmd/tailscale/cli/serve.go#L686

I am using it here and there.

Reply:
🤦‍♂️

Reply:
Wait, no, I mean in this method. Why is this a method and not a function?

Reply:
You're right! 🤦‍♂️ You said receiver, and I 🧠💨 . I moved it to a function and gave it a more specific name since we're in the somewhat sprawling `cli` package.